modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,353 @@
+
+The Mule upgrade tool does not support upgrading clustered Mule instances. For clustered environments, upgrade each of the nodes manually by following the instructions in xref:release-notes::mule-runtime/updating-mule-4-versions.adoc#mulerunvers[Upgrading an On-Premises Mule Instance Managed Through Runtime Manager]
+
+== Before You Begin
+
+Before starting your upgrade, ensure that you have:
+
+* A currently operational Mule 4 instance, in _stopped_ status to prepare for the upgrade
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory
++
+If the Mule upgrade tool is not bundled with your current Mule distribution, download the tool from the https://help.mulesoft.com/s/[Help Center^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.
+* The latest Mule runtime engine distribution, downloaded and unpacked in your system
++
+The Mule upgrade tool does not support upgrading to patch distributions, identified by including the word `patch` in the file name. Ensure you download a full Mule distribution, identified by including the word `distribution` in the file name, for example: `mule-ee-distribution-standalone-4.4.0-20220221.zip`.
++
+You can download Mule distributions from the https://help.mulesoft.com/s/[Help Center^].
+* At least 2 GB of available disk space on the file system and access privileges to install the new Mule distribution
+* (For Windows environments) The execution policy for Powershell scripts set to *Unrestricted*
+
+//This feature is not yet available
+////
+== Upgrade Mule to the Latest Version
+
+To upgrade your Mule instance to the latest version:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade` command:
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade
+```
+////
+
+== Upgrade Mule
+
+To upgrade your Mule instance to the latest version:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade -n` subcommand, specifying as an argument the directory where the latest Mule distribution is downloaded and unpacked:
++
+[source,bash,linenums]
+----
+$MULE_HOME/tools/upgrade-tool upgrade -n ./<newMuleDistributionPath>
+----
+
+== Roll Back an Upgrade
+
+The Mule upgrade tool generates a backup of your existing Mule instance before starting the upgrade process. The backup is stored in the `$MULE_HOME/backup` folder in case you want to use the tool to restore your upgraded Mule instance to its previous state.
+
+To perform a rollback:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `rollback` subcommand:
++
+[source,bash,linenums]
+----
+$MULE_HOME/tools/upgrade-tool rollback
+----
+
+== Troubleshooting File Conflicts
+
+Before performing the upgrade, the tool compares configuration files from the _current_ Mule distribution and the _new_ distribution to detect changes and prevent these changes from being lost.
```

Comment:
"the tool compares configuration files of the _current_ Mule Runtime distribution with the _new_ distribution to detect changes and prevent these changes from being lost.